nss-day-cohort-46/rare-client-groovy-goblins
# Cr create category

# Description

On the categories tab, user can click `Create Category` and is taken to the create form.
User is redirected to the categories endpoint after creating a new category.

Closes #9 

## Type of change


# Testing Instructions

In `client` terminal

`git fetch --all`
`git checkout cr-create-category`
`npm start`

1. In the categories tab, click `Create Category`.
2. Add a new category.
3. User is redirected to the `categories` endpoint where the new category should appear in the list of categories.


# Checklist:
